Sources/DarkModeCore/DMTraitCollection.h

```diff
@@ -15,14 +17,21 @@ typedef NS_ENUM(NSInteger, DMUserInterfaceStyle) {
 
 @interface DMTraitCollection : NSObject
 
-@property (class, nonatomic, strong) DMTraitCollection *currentTraitCollection;
+@property (class, nonatomic, readonly) DMTraitCollection *currentTraitCollection;
+@property (class, nonatomic, readonly) DMTraitCollection *lastManuallySetTraitCollection;
```

Comment:
Does this property need to be public?

Reply:
lastManuallySetTraitCollection corresponds to the UI style user chooses in app (light/dark/follow system(unspecified)), where currentTraitCollection reflects the actual userInterfaceStyle app adopts (only light and dark), so it should be public for query, but a better name might be needed

Reply:
As for naming, how about `overrideTraitCollection` or lastOverrideTraitCollection`? There is an example from Apple: https://developer.apple.com/documentation/uikit/uipresentationcontroller/1618335-overridetraitcollection?language=objc

Similar pattern can be found in Flutter: https://api.flutter.dev/flutter/material/ThemeData/cupertinoOverrideTheme.html

Basically, `manuallySet` can be `override`.

Reply:
Nice! Fixed the naming in the latest commit